server/service/src/asset/update.rs

```diff
@@ -163,6 +164,7 @@ pub fn generate(
     }: UpdateAsset,
     mut asset_row: AssetRow,
 ) -> AssetRow {
+    asset_row.deleted_datetime = None;
```

Comment:
Set this to null anytime it's updated to ensure that the record isn't lost when moved back to a store. We use a separate method to delete...